lib/payment_processor/go_cardless/subscription.rb

```diff
@@ -64,6 +64,7 @@ def action_params
           is_subscription:      true,
           recurrence_number:    0,
           card_expiration_date: nil,
+          mandate_reference:    mandate.reference,
```

Comment:
its the `reference` field of the mandate object, and we have to include it in emails. see [here](https://developer.gocardless.com/2015-07-06/#core-endpoints-mandates)